src/Containers/StoreHoursList/StoreHoursList.tsx

```diff
@@ -0,0 +1,529 @@
+                                        }
+                                        else if (allCategoriesWithHours.length !== 1) { 
+                                            setAllCategoriesWithHours(allCategoriesWithHours.filter((el): ICategoryWithHoursTypes | null | boolean => el.category !==  listAllCategories[1].category)); 
+                                        } 
+                                    }}
+                                >
+                                    { listAllCategories[1].category }
+                                </StyledTag>
+                            )
+                        })}
+                    </TagContainer>
+                    <CenteredButton 
+                        icon={Add} 
+                        onClick={(): void => {
+                            if(input !== null || input !== undefined || input === '') {
+                                const newCategory = createCategoryWithHours(input);
+                                setAllCategoriesWithHours([...allCategoriesWithHours, newCategory]);
+                                setSuccess(true);
+                            }
+                            setEditCategoryModalState(!editCategoryModal);
+                        }}
+                    > 
+                        { constants.BUTTONS.ADD_CATEGORY } 
+                    </CenteredButton> 
+                </StyledModal>
+                <StyledSelect
+                    label={constants.TITLES.CHANGE_ACTIVE}
+                    description={constants.TITLES.CHANGE_ACTIVE_SUBTITLE}
+                    placeholder={activeCategory}
+                    onChange={(e: React.ChangeEvent<HTMLInputElement>): void => {
+                        setSelectActiveCategory(e.target.value);
+                    }}
+                    value={selectActiveCategory}
+                >
+                    {Object.entries(allCategoriesWithHours).map((listAllCategories): React.ReactElement => {
+                        return (
+                            <option
+                                key={listAllCategories[0]} 
+                                value={listAllCategories[1].category}
+                            >
+                                { listAllCategories[1].category}
+                            </option>
+                        )
+                    })}
+                </StyledSelect>
+                <ButtonsContainer>
+                    <StyledButton 
+                        onClick={(): void => {
+                            setActiveCategorySchedule(getActiveSchedule(selectActiveCategory));
+                            setError('');
+                        }}
+                    > 
+                        { constants.BUTTONS.SET_ACTIVE }
+                    </StyledButton>
+                </ButtonsContainer>        
+            </StyledModal>
+            <StyledModal state={addModal}>
```

Comment:
This should be a component in itself

Reply:
@fpunny Isn't this already a Modal component, it's just styling being applied? Or are you saying to make this a component with the styling preset?